examples/styletron/app/styletron.ts

```diff
@@ -0,0 +1,35 @@
+import { Client, Server } from "styletron-engine-atomic"; // or "styletron-engine-monolithic"
+
+/**
+ * The Styletron engine to use on the current runtime.
+ */
+export const styletron =
+  typeof window === "undefined"
```

Comment:
```suggestion
  typeof document === "undefined"
```

For deno support, see https://remix.run/docs/en/v1/pages/gotchas#typeof-window-checks